optimade/models/structures.py

```diff
@@ -307,7 +313,7 @@ class StructureResourceAttributes(EntryResourceAttributes):
   - A filter that matches an exactly given formula is :filter:`chemical_formula_anonymous="A2B"`.""",
     )
 
-    dimension_types: conlist(len_eq=3) = Field(
+    dimension_types: List[Periodicity] = Field(
```

Comment:
Shouldn't this be `Tuple[Periodicity, Periodicity, Periodicity]` which would eliminate the need for `min_items` and `max_items`?